megamek/src/megamek/client/ui/swing/util/TurnTimer.java

```diff
@@ -60,6 +62,8 @@ public void actionPerformed(ActionEvent ae) {
                 remaining.setText(minutes+":"+seconds);
                 progressBar.setValue(counter);
                 if (counter<1) {
+                    // prevent the popup dialog from breaking time limit
+                    GUIPreferences.getInstance().setNagForNoAction(false);
```

Comment:
This is a global setting that persists across sessions and should not be changed by the settings for a particular game. Even within the same game, it will cause it not to show if the user clicks done before the timer runs out, and it should show for that. The best solution I can think of is to add a boolean parameter to AbstractPhaseDisplay#ready that can be used to disable the nag popup.

Reply:
Well, or I could disable it, call ready() and enable it again, if it was enabled before
Whichever you prefer